### ISSUE TITLE
chown: cannot read directory ‘/var/lib/sentry/files’: Permission denied

### ISSUE THREAD

**Issue by fuhongwei041** [getsentry/docker-sentry#109]
i encounter this problem. can you help me?
```
root@localhost onpremise]# docker-compose run --rm web config generate-config-key
Starting onpremise_memcached_1 ... 
Starting onpremise_postgres_1 ... 
Starting onpremise_smtp_1 ... 
Starting onpremise_redis_1 ... done
chown: cannot read directory ‘/var/lib/sentry/files’: Permission denied
```
version: '2'
services:
  base:
    build: .
    environment:
      # Run `docker-compose run web config generate-secret-key`
      # to get the SENTRY_SECRET_KEY value.
      SENTRY_SECRET_KEY: '#*r@jgcg5h6e_kfxfheqgf+2^cqky24!%0wal(a1)72+ozf%m3'
      SENTRY_MEMCACHED_HOST: memcached
      SENTRY_REDIS_HOST: redis
      SENTRY_POSTGRES_HOST: postgres
      SENTRY_EMAIL_HOST: smtp
    volumes:
      **- /opt/sentry/onpremise/data/sentry:/var/lib/sentry/files**

**Comment by fuhongwei041** [comment]
is there missing USER sentry ?
--
FROM python:2.7.13-slim
  |  
  | # add our user and group first to make sure their IDs get assigned consistently
  | RUN groupadd -r sentry && useradd -r -m -g sentry sentry
  |  
  | RUN apt-get update && apt-get install -y --no-install-recommends \
  | gcc \
  | git \
  | libffi-dev \
  | libjpeg-dev \
  | libpq-dev \
  | libxml2-dev \
  | libxslt-dev \
  | libyaml-dev \
  | && rm -rf /var/lib/apt/lists/*

**Comment by fuhongwei041** [comment]
fixed after i upgrade docker from 1.12 to 17.06.0-ce
```
[root@localhost sentry]# docker version
Client:
 Version:      17.06.0-ce
 API version:  1.30
 Go version:   go1.8.3
 Git commit:   02c1d87
 Built:        Fri Jun 23 21:20:36 2017
 OS/Arch:      linux/amd64

Server:
 Version:      17.06.0-ce
 API version:  1.30 (minimum version 1.12)
 Go version:   go1.8.3
 Git commit:   02c1d87
 Built:        Fri Jun 23 21:21:56 2017
 OS/Arch:      linux/amd64
 Experimental: false

```
